<!doctype html>
<html>
  <head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="chrome=1">
    <title>DFLpred by fanchi</title>

    <link rel="stylesheet" href="stylesheets/styles.css">
    <link rel="stylesheet" href="stylesheets/github-light.css">
    <meta name="viewport" content="width=device-width">
    <!--[if lt IE 9]>
    <script src="//html5shiv.googlecode.com/svn/trunk/html5.js"></script>
    <![endif]-->
  </head>
  <body>
    <div class="wrapper">
      <header>
        <h1>DFLpred</h1>
        <p>High-throughput prediction of disordered flexible linker regions in protein sequences</p>

        <p class="view"><a href="https://github.com/fanchi/DFLpred">View the Project on GitHub <small>fanchi/DFLpred</small></a></p>


        <ul>
          <li><a href="https://github.com/fanchi/DFLpred/zipball/master">Download <strong>ZIP File</strong></a></li>
          <li><a href="https://github.com/fanchi/DFLpred/tarball/master">Download <strong>TAR Ball</strong></a></li>
          <li><a href="https://github.com/fanchi/DFLpred">View On <strong>GitHub</strong></a></li>
        </ul>
      </header>
      <section>
        <h1>
<a id="dflpred" class="anchor" href="#dflpred" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a>DFLpred</h1>

<p>DFLpred is designed for the prediction of disordered flexible linker (DFL). The method generates numeric score for each residue in the input protein sequence that quantifies putative propensity to form a DFL. Larger value of propensity denotes higher likelihood to form DFL. It also provides putative binary annotations (a given residue is predicted either as a DFL or not a DFL) based on false positive rate = 0.05 using threshold on the propensity score = 0.18). Residues with propensity &gt; 0.18 are assumed to form DFLs and otherwise they are assumed not to form DFLs.  </p>

<h1>
<a id="system-requirement" class="anchor" href="#system-requirement" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a>System requirement</h1>

<p>Java Runtime Environment 1.5 or later</p>

<h1>
<a id="usage-command-line" class="anchor" href="#usage-command-line" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a>Usage (command line)</h1>

<p>java -jar DFLpred.jar [input_fasta_file] [output_file]<br>
Accepts up to 5000 FASTA sequences in a single fasta file</p>

<h1>
<a id="example" class="anchor" href="#example" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a>Example</h1>

<p>java -jar DFLpred.jar examples.fasta results.txt</p>

<h1>
<a id="format-of-results" class="anchor" href="#format-of-results" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a>Format of results</h1>

<p>Line 1: &gt;protein ID<br>
Line 2: protein sequence using 1-letter amino acid encoding. Lower/upper case indicates a residue is/is not predicted as a flexible disordered linker (DFL) by the score threshold 0.18.<br>
Line 3: comma-separated propensity score of each residue to be a DFL.</p>

<h1>
<a id="dflpred-webserver" class="anchor" href="#dflpred-webserver" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a>DFLpred webserver</h1>

<p><a href="http://biomine-ws.ece.ualberta.ca/DFLpred">http://biomine-ws.ece.ualberta.ca/DFLpred</a></p>

<h1>
<a id="citation" class="anchor" href="#citation" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a>Citation</h1>

<p>Meng, F. and Kurgan, L. DFLpred: High-throughput prediction of disordered flexible linker regions in protein sequences. Bioinformatics 2016;32(12):i341-i350.</p>

<h1>
<a id="acknowledgments" class="anchor" href="#acknowledgments" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a>Acknowledgments</h1>

<p>We acknowledge with thanks the following software used as a part of DFLpred:<br>
    IUPred (<a href="http://iupred.enzim.hu/">http://iupred.enzim.hu/</a>) - Prediction of Intrinsically Unstructured Proteins<br>
    AAindex (<a href="http://www.genome.jp/aaindex/">http://www.genome.jp/aaindex/</a>) - Amino acid indices, substitution matrices and pair-wise contact potentials  </p>

<h1>
<a id="disclaimer-and-license-agreement" class="anchor" href="#disclaimer-and-license-agreement" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a>Disclaimer and license agreement</h1>

<p>see <a href="http://biomine-ws.ece.ualberta.ca/Disclaimer.html">http://biomine-ws.ece.ualberta.ca/Disclaimer.html</a></p>

<h1>
<a id="biomine-home-page" class="anchor" href="#biomine-home-page" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a>Biomine home page</h1>

<pre><code>http://biomine.ece.ualberta.ca/
</code></pre>
      </section>
      <footer>
        <p>This project is maintained by <a href="https://github.com/fanchi">fanchi</a></p>
        <p><small>Hosted on GitHub Pages &mdash; Theme by <a href="https://github.com/orderedlist">orderedlist</a></small></p>
      </footer>
    </div>
    <script src="javascripts/scale.fix.js"></script>
    
  </body>
</html>
